core/src/worker/workflow/machines/update_state_machine.rs

```diff
@@ -75,6 +75,7 @@ pub(super) struct SharedState {
     instance_id: String,
     event_seq_id: i64,
     request: UpdateRequest,
+    replaying_when_created: bool,
```

Comment:
Can you explain what the "when_created" part of the name means?

Reply:
When the state machine is created